### ISSUE TITLE
Screen reader support is lacking

### ISSUE THREAD

**Issue by yuriy-fix** [vaadin/vaadin-list-box#50]
The component (including the items in the dropdown) is announced as a generic “group”.
The changes to the selection are not announced.

**Comment by anezthes** [comment]
Tested in Chrome and Safari with following structure:
**vaadin-list-box**
```
<ul role="listbox">
  <slot></slot>
</ul>
```
**vaadin-item**
```
<li id="focusElement" role="option" tabindex="-1">
  <slot></slot>
</li>
```
Manage the focus and tab indices and you're good to go.

**Comment by web-padawan** [comment]
Closing in favor of https://github.com/vaadin/vaadin-web-components/issues/154 which describes the same problem in more detail.
